package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "cf-app-a-better-browser",
+  "version": "2.0.0",
+  "description": "Warns users with a simple message at the top of your website if their browser version is too old.",
+  "scripts": {
+    "start": "webpack --watch --colors",
+    "build": "NODE_ENV=production webpack --define process.env.NODE_ENV=\"'production'\" --colors"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xPaw/CF-ABetterBrowser.git"
+  },
+  "private": true,
+  "contributors": [
+    {
+      "name": "Pavel Djundik",
+      "email": "abetterbrowser@xpaw.me"
+    },
+    {
+      "name": "Eugene Rodionov",
+      "email": "abetterbrowser@theaqua.im"
+    }
+  ],
+  "dependencies": {
+    "detect-browser": "^1.6.2"
+  },
+  "devDependencies": {
+    "autoprefixer": "^6.7.7",
+    "babel-core": "^6.24.0",
+    "babel-eslint": "^7.1.1",
+    "babel-loader": "^6.4.1",
+    "babel-preset-env": "^1.2.2",
+    "css-loader": "^0.27.3",
+    "extract-loader": "^0.1.0",
+    "file-loader": "^0.10.1",
+    "postcss-loader": "^1.3.3",
+    "standard": "^9.0.1",
+    "standard-loader": "^6.0.1",
+    "style-loader": "^0.14.1",
+    "webpack": "^2.2.1"
+  },
+  "standard": {
```

Comment:
This `"standard"` property lists some [common environment variables](https://cloudflareapps.com/apps/developer/docs/writing-your-app/environment) that are injected into the app during runtime. Normally these would be flagged as undeclared globals by linters.